src/chat/CustomerSupport/Domain.cs

```diff
@@ -1,13 +1,16 @@
 ﻿using System.Text.Json.Serialization;
-using Microsoft.Extensions.AI;
 
 public class ManualChunk
 {
     public int ChunkId { get; set; }
+
     public int ProductId { get; set; }
+
     public int PageNumber { get; set; }
+
     public required string Text { get; set; }
-    public required Embedding<float> Embedding { get; set; }
+
+    public required ReadOnlyMemory<float> Embedding { get; set; }
```

Comment:
@luisquintanilla - looking back at this code, why do we move away from an Embedding<T> type to ReadOnlyMemory<T>? The Embedding type feels like it helps clarify what this is